cassandane/Cassandane/PortManager.pm

```diff
@@ -58,7 +58,7 @@ sub alloc
             if (defined($workerid) && $workerid eq 'invalid');
         my $cassini = Cassandane::Cassini->instance();
         my $cassini_base_port = $cassini->val('cassandane', 'base_port') // 0;
-        $base_port = 0 + $cassini_base_port || 9100;
+        $base_port = 0 + $cassini_base_port || 29100;
```

Comment:
Heh... `$cassini->val()` takes an optional third argument, the default value to use if no value was configured.  So these two lines could have been one all along, like:

```
$base_port = 0 + $cassini->val('cassandane', 'base_port', '[2]9100');
```

Reply:
Yeah!  Done :)